Solutions/CohesitySecurity/Playbooks/Cohesity_Delete_Incident_Blobs/readme.md

```diff
@@ -0,0 +1,40 @@
+# Cohesity Delete incident blobs.
+## Summary
+This playbook deletes the blobs on Azure storage created by an incident that is generated by Cohesity function apps. For example, an automation rule can be created to delete the blobs used for an incident by running this playbook when the corresponding Sentinel ticket is closed.
+
+## Deployment instructions
+1. Click on the "Deploy to Azure" button to deploy the playbook. This step directs you to deploy an ARM Template wizard.
```

Comment:
When I click the Deploy to Azure button, I'm getting 

> There was an error downloading the template from URI 'https://raw.githubusercontent.com/cohesity/Azure-Sentinel/CohesitySecurity.internal/Solutions/CohesitySecurity/Playbooks/Cohesity_Delete_Incidient_Blobs/azuredeploy.json'. Ensure that the template is publicly accessible and that the publisher has enabled CORS policy on the endpoint. To deploy this template, download the template manually and paste the contents in the 'Build your own template in the editor' option below. 

Please fix.

Reply:
Done.